python/tank/platform/engine.py

```diff
@@ -289,6 +296,43 @@ def clear_busy(self):
         if self.__global_progress_widget:
             self.execute_in_main_thread(self.__clear_busy)
 
+    def log_metric(self, action):
+        """Log an engine metric.
+
+        :param action: Action string to log, e.g. 'Init'
+
+        Logs a user activity metric as performed within an engine. This is
+        a convenience method that auto-populates the module portion of
+        `tank.util.log_user_activity_metric()`
+
+        Internal Use Only - We provide no guarantees that this method
+        will be backwards compatible.
+
+        """
+
+        # the action contains the engine and app name, e.g.
+        # module: tk-maya
+        # action: tk-maya - Init
+        full_action = "%s - %s" % (self.name, action)
```

Comment:
minor: If we are dropping the dash for apps, maybe do it consistently here as well?